_posts/2022-06-23-cockpit-272.md

```diff
@@ -0,0 +1,69 @@
+---
+title: Cockpit 272
+author: garrett
+date: '2022-06-23'
+tags: cockpit
+slug: cockpit-272
+category: release
+summary: 'Edit custom firewall ports, pin favorite services, dark login support, and unprivilaged cockpit/ws mode'
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/).
+We release regularly.
+
+Here are the release notes from Cockpit 272:
+
+## Firewall: Edit custom services
+
+The Firewall page has been able to open custom ports by adding creating custom services. This release adds the ability to edit custom services.
+
+![edit-custom]({{ site.baseurl }}/images/272-firewall-edit-custom-services.png)
+
+## Services: Pin services as favorites
+
+The services page now allows users to pin any service to the top of the services list.
+
+![pin-2]({{ site.baseurl }}/images/272-services-pin-services-as-favorites.png)
+
+To pin a service, navigate to its detail page and click `Pin unit` in the menu next to its name.
```

Comment:
Should we use monospace "code" attribute for something that's not monospace in the UI?

Reply:
```suggestion
To pin a service, navigate to its detail page and click "Pin unit" in the menu next to its name.
```